guides/stable/3D_interactivity.md

```diff
@@ -0,0 +1,131 @@
+# 3D interactivity
+
+## Coordinate systems in napari
+In napari, there are three main coordinate systems: (1) canvas, (2) world, and (3) layer. The canvas coordinates system is the 2D coordinate system of the canvas on which the scene is rendered. World coordinates are the nD coordinates of the entire scene. As the name suggests, the layer coordinates is the nD coordinate system of the data in a given layer. Layer coordinates are specific the each layer's data and are related to the world coordinate system via the layer transforms.
+
+![coordinate-systems](images/3d_interaction_coordianates.png)
+
+## In 3D mode, clicks are lines
+Since the 3D scene is rendered on a 2D surface (your screen), your mouse click does not map to a specific point in space. Thus, when rendering a scene in 3D, napari interprets your click as a line in the direction of the camera view that goes through the point that view ray intersects the camera plane.
+
+![click-line](images/3d_interaction_click_line.png)
+
+When a user clicks or moves the mouse in the canvas, napari emits a mouse event with the following properties:
+
+
+- pos: the position of the click in canvas coordinates.
+- position: the position of the click in world coordinates.
```

Comment:
```suggestion
- world_position: the position of the click in world coordinates.
```
What does this variable contain? Above you mentioned that this cannot be a point. What is it then?

Reply:
I will clarify the description of `pos` and `position` in the docs. I agree that both the names and descriptions are unclear. As @alisterburt  said, the names are pre-existing napari public API, so I think we should discuss in a separate issue.